docs/running from source.md

```diff
@@ -0,0 +1,53 @@
+# Running From Source
+
+If you just want to play and there is a compiled version available on the
+[Archipelago releases page](https://github.com/ArchipelagoMW/Archipelago/releases),
+use that version. These steps are for developers or platforms without compiled releases available.
+
+## General
+
+What you'll need:
+ * Python 3.8.7 or newer
```

Comment:
We should probably remove Python 3.8.7 since support for it will be dropped relatively soonish or at the very least "suggest" getting the latest stable release

Reply:
currently that's the minimum required version and py4a land (potential Android support) is still on 3.8, so until we have a definite answer, I'd like it to be accurate.